src/components/DemoConfigDialog/DemoConfigDialog.tsx

```diff
@@ -136,16 +137,24 @@ const DemoConfigDialog = ({ selectedConfigSource, configQuery }: Props) => {
 
   return (
     <>
+      {!configQuery.isSuccess && (
+        <Helmet>
+          <meta name="description" content="Enter the App Config ID" />
+        </Helmet>
+      )}
       {configQuery.isSuccess && (
         <div className={styles.note}>
           <div>{t('currently_previewing_config', { configSource: selectedConfigSource })}</div>
-          <Link onClick={clearConfig}>{t('click_to_unselect_config')}</Link>
+          <Link href="javascript:;" onClick={clearConfig}>
```

Comment:
Maybe we should just make this a Button with variant = text?

Reply:
Fixed it with `variant=text`, though styling changed a bit.

Reply:
Specific styling on that was never anything intentional. I think it looks fine.